### DISCUSSION TITLE
mount.mergerfs not included in static binaries releases

### DISCUSSION THREAD

**Discussion by lucsansag** [trapexit/mergerfs#1359] [Q&A]
Hi, I just migrated to mergerfs from mhddfs today and after getting it working I thought about creating a new mergerfs-bin AUR package, since compiling it from source took a while on my Raspberry Pi 4. However when I took a look at the latest release's static binaries for arm64 I realized `mount.mergerfs` wasn't included. According to the README, I guess I could probably depend on fuse2 and use `fuse.mergerfs` in the fstab instead of just `mergerfs` but I was wondering, why is it not included?

**Comment by trapexit** [comment]
Probably overlooked. I'll take a look into it when I get the time.
